var/spack/repos/builtin/packages/musl/package.py

```diff
@@ -31,10 +31,29 @@ class Musl(Package):
     version('1.1.21', sha256='c742b66f6f49c9e5f52f64d8b79fecb5a0f6e0203fca176c70ca20f6be285f44')
     version('1.1.20', sha256='44be8771d0e6c6b5f82dd15662eb2957c9a3173a19a8b49966ac0542bbd40d61')
 
-    phases = ['configure', 'build', 'install']
+    phases = ['edit', 'configure', 'build', 'install']
```

Comment:
If you want to keep the phases the same, you could rename `edit()` to `patch()`.